src/rt/sections_android.d

```diff
@@ -63,11 +63,11 @@ void initSections() nothrow @nogc
 {
     pthread_key_create(&_tlsKey, null);
 
-    auto mbeg = cast(immutable ModuleInfo**)&__start_minfo;
-    auto mend = cast(immutable ModuleInfo**)&__stop_minfo;
+    auto mbeg = cast(immutable ModuleInfo**)&__start___minfo;
+    auto mend = cast(immutable ModuleInfo**)&__stop___minfo;
```

Comment:
Thanks for the link.  I thought a `grep` of the linker scripts would turn them up, but that's obviously not going to work for a section name we make up.